yt_dlp/extractor/twitter.py

```diff
@@ -1231,6 +1234,11 @@ def _real_extract(self, url):
 
         def extract_from_video_info(media):
             media_id = traverse_obj(media, 'id_str', 'id', expected_type=str_or_none)
+            if not media_id:
+                # Workaround for syndication fallback
+                media_id = traverse_obj(media, (
+                    'video_info', 'variants', ..., 'url',
+                    {functools.partial(re.search, r'_video/(\d+)/')}, 1), get_all=False)
```

Comment:
```suggestion
            media_id = traverse_obj(media, 'id_str', 'id', (
                'video_info', 'variants', ..., 'url',
                {functools.partial(re.search, r'_video/(\d+)/')}, 1
            ), get_all=False, expected_type=str_or_none)
```
(formatting may need change)